### ISSUE TITLE
securedrop-app-code build process is downloading wheels

### ISSUE THREAD

**Issue by redshiftzero** [freedomofpress/securedrop#4755]
## Description

the current build process is downloading wheels during the install of the pip requirements in the build container

## Steps to Reproduce

Run the command corresponding to the task: `Install SecureDrop Python requirements in container`:

```
pip3 install --require-hashes -r <path_to_requirements.txt>
```

## Expected Behavior

no wheels are downloaded, only source tarballs

## Actual Behavior

wheels are downloaded

## Comments

adding `--no-binary :all:` to the pip install call should resolve

**Comment by redshiftzero** [comment]
so there's a fix in 21ee63400f347ea641330e31d2f3f583e53840fa but @rmol had a smart idea which is to add additional verbosity/debug logging such that the person doing the build can more easily verify that wheels are not being download when invoking `make build-debs`, let's add that along with the above commit in a new PR

**Comment by kushaldas** [comment]
This will have to be removed when we will move to make the package reproducible (the `dh-virtualenv` part).
